Tests/FluentAssertions.Specs/Numeric/NullableNumericAssertionSpecs.BeGreaterThan.cs

```diff
@@ -67,5 +67,135 @@ public void NaN_is_never_greater_than_another_double()
                 .Should().Throw<XunitException>()
                 .WithMessage("*NaN*");
         }
+
+        [Theory]
+        [InlineData(5, 5)]
+        [InlineData(1, 10)]
+        [InlineData(0, 5)]
+        [InlineData(0, 0)]
+        [InlineData(-1, 5)]
+        [InlineData(-1, -1)]
+        [InlineData(10, 10)]
+        public void To_test_the_null_path_for_difference_on_nullable_int(int? subject, int expectation)
+        {
+            // Arrange
+            // Act
+            Action act = () => subject.Should().BeGreaterThan(expectation);
+
+            // Assert
+            act
+                .Should().Throw<XunitException>()
+                .Which.Message.Should().NotMatch("*(difference of 0)*");
```

Comment:
0% difficult :D

Reply:
Hmm.. actually pretty difficult, because we want to know if something is absent (the `difference of..` part)

Reply:
That could be a feature request: `WithoutMessage()` or similar

Reply:
> Hmm.. actually pretty difficult, because we want to know if something is absent (the difference of 0 part).

I don't get the purpose of the test. Why would we check the absence of `difference of 0`?

Reply:
Or we could just check if the test throws, but I feel if this can lead to false positive.

Reply:
https://github.com/fluentassertions/fluentassertions/pull/2686#discussion_r1653504172

@dennisdoomen @jnyrup Can you follow this rational?

Reply:
I get the purpose now.